### ISSUE TITLE
How should Voting on nominations take place

### ISSUE THREAD

**Issue by adammatlack** [ioncoincore/ion#9]
Community control nominations are forming. We now need to decide how these nominations will be voted on and finalized for appointment.

My current suggestion with sentinel disabled is we leverage ATP.

Once a nomination gets enough support we create a token that represents a vote for that user.

The goal should be at a minimum of 3 users to control/have write access. 

Example of how this will work is as follows:

5 users are nominated and seconded.  Users A, B, C, D, E.

We create 5 tokens representing each user for every ion in existence. We then take a snapshot at a designated time on all balances over 100 ion. We then air drop a 1 to 1 to all ion holders. We then create a yes address and no address. And users can then cast their vote. We set a week time limit on voting (from the start of voting) and a majority is required to elect a nominee.

Any thoughts?

**Comment by cryptolinux** [comment]
@adammatlack here https://github.com/ioncoincore/ion/projects/1

**Comment by Cryptoformer** [comment]
I would like to make an addition to that, voting does only include a specific role, no generic permissions to anyone. This didn't work in the past and shouln't continued. Can we make a sub-task out of it to define roles first?

**Comment by adammatlack** [comment]
A List of roles would be helpful for the nomination process then. If anyone wants to provide their ideal list that would be helpful.

**Comment by cryptolinux** [comment]
yes, thats why I asked to create cards about it (or issues in your current case which somebody has to manage and assign).

This issue open, and we should discuss it and which roles there could be and actually a must to have. Finally, we can write a result of current issue to: #10 where this discussion would stay linked which would explain how and why those roles were defined.

**Comment by korvasc** [comment]
I think this concept is brilliant and makes for a great use case example. For this, though I'm not so sure as candidates will ideally need to be completely independent of Ionomy so I don't think that the voting pool will be big enough

**Comment by Cryptoformer** [comment]
I would like to make an addition to the voting system, we need one token for each role which we are about to define, we could also make a voting on the role itself to get approval. 

Question is would we like to have a minimum quorum to get approval for a sucessful vote or do we dismiss that?

**Comment by adammatlack** [comment]
i think there needs to be a percentage that has to be reached i would say at a minimum 10%
